vip_hci/phot/fakecomp.py

```diff
@@ -18,16 +17,22 @@
 from ..preproc import cube_crop_frames, frame_shift, frame_crop
 from ..var import frame_center, fit_2dgaussian, get_circle
 
+def cube_shift(cube, y, x, imlib):
```

Comment:
Not sure we need a function for something that could take a single line. Anyway let's leave it, but put it at the end of the file, and complete de documentation